src/content/learn/thinking-in-react.md

```diff
@@ -25,59 +25,59 @@ The JSON API returns some data that looks like this:
 ]
 ```
 
-The mockup looks like this:
+Rancang bangun kita terlihat seperti ini:
```

Comment:
Similar to the previous sentence. Also, the source is not `Our mockup`, but `The mockup`.

```suggestion
Rancang bangun tersebut terlihat seperti ini:
```